css/style.css

```diff
@@ -21,4 +21,5 @@
     align-items: center;
     color: black;
     font-family: 'Champions Light';
-  }
```

Comment:
I see that already in line 24, after adding the font type, you finished your style sheet, I reviewed the final project and I think that you should add some things in your CSS file to be able to pass the challenge correctly, here are some suggestions that you can put to make your page look better and more similar to the final project:

-I think you can use the Background-color property to modify the background color of <main> tag and <figure> tag to make them more similar to the final project.

-You could use the padding:, margin: or text-align: properties to get them positioned as in the final requested project.